keel-core/src/main/kotlin/com/netflix/spinnaker/keel/veto/unhappy/UnhappyVeto.kt

```diff
@@ -39,7 +38,6 @@ import org.springframework.stereotype.Component
  */
 @Component
 class UnhappyVeto(
-  private val resourceRepository: ResourceRepository,
```

Comment:
was this just leftover and not used anymore?

Reply:
Correct.